pkg/utils/boards/boards.go

```diff
@@ -0,0 +1,99 @@
+package boards
+
+import (
+	"fmt"
+	"github.com/joho/godotenv"
+	"github.com/kairos-io/kairos-agent/v2/pkg/partitioner"
+	v1 "github.com/kairos-io/kairos-agent/v2/pkg/types/v1"
+	"github.com/kairos-io/kairos-agent/v2/pkg/utils/partitions"
+	"github.com/sanity-io/litter"
+	"os"
+
+	cnst "github.com/kairos-io/kairos-agent/v2/pkg/constants"
+)
+
+// IsAnAndroidBoard returns true if the system is an Android board
+// based on checking if there is a build.prop file
+func IsAnAndroidBoard() bool {
+	// Check if we are running on an Android board
+	_, err := os.Stat("/build.prop")
+	if err == nil {
+		return true
+	}
+	return false
+}
+
+// GetAndroidBoardModel returns the board model if the system is an Android board
+func GetAndroidBoardModel() string {
+	// Check if we are running on an Android board
+	if IsAnAndroidBoard() {
+		buildProp, err := godotenv.Read("/build.prop")
+		if err != nil {
+			return ""
+		}
+		switch buildProp["ro.product.board"] {
```

Comment:
I made this generic, but could easily merge this and the above function into one to check just for this board. thoughts?

Reply:
I think this is much better than just checking for this board, that way if tomorrow we for any reason have another board, this is there and we avoid a quick and dirty :P